pkg/apis/apps.kubermatic/v1/application_installation.go

```diff
@@ -125,8 +125,9 @@ type Version struct {
 // ApplicationInstallationStatus denotes status information about an ApplicationInstallation.
 type ApplicationInstallationStatus struct {
 	LastUpdated metav1.Time `json:"lastUpdated,omitempty"`
```

Comment:
@SimonTheLeg i think we can remove this field as condition as `LastTransitionTime` and  `LastHeartbeatTime`. field. WDYT?

Reply:
as discussed in meeting we can remove this field